docs/streamtypes/caip-10-link/api.md

```diff
@@ -0,0 +1,126 @@
+
+[:octicons-file-code-16: API reference](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_stream_caip10_link.caip10link-1.html#fromAccount){:target="_blank"}
+
+
+### Parameters
+
+#### ceramic
+
+
+When creating or querying a Caip10Link, the first parameter is the `CeramicAPI` used to communicate with the ceramic node and it is always required. It will either be an instance of [`Ceramic`](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_core.ceramic.html){:target="_blank"} when using the Core client or an instance of [`CeramicClient`](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_http_client.ceramicclient.html){:target="_blank"} when using the HTTP client.
+
+#### accountId
+
+The blockchain account ID - in Caip10 format - of the blockchain account being linked.
+
+#### opts (optional)
+The final argument to `Caip10Link.fromAccount` is an instance of [`CreateOpts`](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_common.createopts-1.html){:target="_blank"}, which are options that control network behaviors performed as part of the operation.  They are not included in the link itself.
+
+| Parameter     | Required?   | Value            | Description | Default value |
+| ------------- | ----------- | ---------------- | ----------- | ----- |
+| `anchor`      | optional    | boolean          | Request an anchor after creating the link | true |
+| `publish`     | optional    | boolean          | Publish the new link to the network | true |
+| `sync`        | optional    | enum             | Controls behavior related to syncing the current link state from the network | SyncOptions.PREFER_CACHE |
+| `syncTimeoutSeconds` | optional    | number            | How long to wait to hear about the current state of the link from the network | 3 |
+
+[:octicons-file-code-16: API reference](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_common.createopts-1.html){:target="_blank"}
+
+
+## Associate a DID with a Caip10Link
+Use the [`link.setDid()`](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_stream_caip10_link.caip10link-1.html#setdid){:target="_blank"} method to update the DID associated with a given Caip10Link
+
+```javascript
+const link = await Caip10Link.fromAccount(ceramic, accountId, opts)
+await link.setDid(did, authProvider, opts)
+```
+
+**Example**
+
+In this example we create a Caip10Link for the ethereum mainnet account *0x0544dcf4fce959c6c4f3b7530190cb5e1bd67cb8* and then associate it with the DID *did:3:k2t6wyfsu4pg0t2n4j8ms3s33xsgqjhtto04mvq8w5a2v5xo48idyz38l7ydki*
+
+```javascript
+import { Caip10Link } from '@ceramicnetwork/stream-caip10-link'
+import { EthereumAuthProvider } from '@ceramicnetwork/blockchain-utils-linking'
+
+const ethProvider = // [...] An ethereum provider for communicating with the ethereum blockchain
```

Comment:
I would just use `window.ethereum` as an example here as that's what most devs will be familiar with.